notifications_api_common/kanalen.py

```diff
@@ -39,7 +47,28 @@ def __repr__(self):
             self.main_resource,
         )
 
-    def get_kenmerken(self, obj: Model, data: Dict = None) -> Dict:
+    @staticmethod
+    def get_field(model: Model, field: str) -> Field:
+        """
+        Function to retrieve a field from a Model
+        """
+        return model._meta.get_field(field)
+
+    def get_help_text(self, field: Field, kenmerk: str) -> str:
+        """
+        Retrieve the help_text for a kenmerk, pulled from the model field by default,
+        but can be overridden by setting extra_kwargs on `Kanaal.__init__`
+        """
+        if help_text := self.extra_kwargs.get(kenmerk, {}).get("help_text"):
+            return help_text
+        return field.help_text
+
+    def get_kenmerken(
+        self,
+        obj: Model,
+        data: Union[Dict, None] = None,
+        request: Union[Request, None] = None,  # noqa
```

Comment:
What does this  request do?

Reply:
Could this just not be in data?

edit: actually probably not

Reply:
Actually again this is something fore open zaak?

Reply:
You're right, this is necessary for Open Zaak to create the URLs: https://github.com/open-zaak/open-zaak/pull/1757/files#diff-b74ad95db3a00961f14403f57ab65597401cc8b8b1ef88e162d3214c4baf70d6R54.

`data` is the actual request data (JSON) that is posted, so it can't really be a part of that I think